scene/resources/2d/tile_set.cpp

```diff
@@ -4943,14 +4946,15 @@ void TileSetAtlasSource::_get_property_list(List<PropertyInfo> *p_list) const {
 				if (default_value.get_type() != Variant::NIL && bool(Variant::evaluate(Variant::OP_EQUAL, value, default_value))) {
 					alternative_property_info.usage ^= PROPERTY_USAGE_STORAGE;
 				}
-				alternative_property_info.name = vformat("%s/%s", vformat("%d", E_alternative.key), alternative_property_info.name);
+				alternative_property_info.name = alternative_property_info_prefix + alternative_property_info.name;
 				tile_property_list.push_back(alternative_property_info);
 			}
 		}
 
 		// Add all alternative.
+		const String property_info_prefix = vformat("%d:%d", E_tile.key.x, E_tile.key.y) + '/';
```

Comment:
```suggestion
		const String property_info_prefix = vformat("%d/%d/", E_tile.key.x, E_tile.key.y);
```

- Copy-paste error.
- Plus `char` is unnecessary since we're already doing `vformat()`.

Reply:
> * Copy-paste error.

I think it should be `%d:%d/%s` with the original code, have I missed something with `vformat`?

Reply:
Ah yes, I looked at the wrong `vformat()` call.

Reply:
I mean, it's not a copy-paste error, using `:` is correct.

Reply:
I know, just pull to local and merge these commits, will fix it.